roles/kubespray-defaults/defaults/main/main.yml

```diff
@@ -254,7 +254,7 @@ kube_apiserver_port: 6443
 
 # If non-empty, will use this string as identification instead of the actual hostname
 kube_override_hostname: >-
-  {%- if cloud_provider is defined and cloud_provider in ['aws'] -%}
+  {%- if cloud_provider is defined -%}
```

Comment:
Same as above (ok, so we have redundant default in the other files, ok :smile: )

Reply:
Would it be better to remove `kube_override_hostname` from the other files and only reserve this one?

Reply:
Yes, absolutely.